### PR TITLE
Update YandexGPT API

### PR DESCRIPTION
**Pull request by tyumentsev4**
Update LLMand Chat model to use new api version

### CODE REVIEW

File: libs/community/langchain_community/chat_models/yandex.py [hunk 1]
```diff
@@ -32,16 +32,15 @@ def _parse_chat_history(history: List[BaseMessage]) -> Tuple[List[Dict[str, str]
...
     for message in history:
         content = cast(str, message.content)
         if isinstance(message, HumanMessage):
             chat_history.append(_parse_message("user", content))
         if isinstance(message, AIMessage):
             chat_history.append(_parse_message("assistant", content))
         if isinstance(message, SystemMessage):
-            instruction = content
-    return chat_history, instruction
+            chat_history.append(_parse_message("system", content))
```
Comment (leo-gan) [inline libs/community/langchain_community/chat_models/yandex.py:42]: Hi @tyumentsev4 Is it possible to add ut-s? The changes are quite heavy.

Comment (tyumentsev4) [reply libs/community/langchain_community/chat_models/yandex.py:42]: Hi @leo-gan, this pull request proposes to stop using the deprecated API v1alpha. Go to use  [Foundation Models API](https://cloud.yandex.com/en/docs/yandexgpt/api-ref/v1/grpc/).

Comment (leo-gan) [reply libs/community/langchain_community/chat_models/yandex.py:42]: Got it. Then it is ok.

**Comment by baskaryan** [comment]
thanks @tyumentsev4!

**Pull request merged**